libraries/node-core-library/src/Import.ts

```diff
@@ -1,13 +1,58 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+import * as nodeJsPath from 'path';
```

Comment:
It's weird not to call it `path`. If it conflicts with another name in this file, we should probably rename the other one.

Reply:
This is what we called it for `FileSystem`, but I'll change it back.